flow-typed/npm/typescript_v3.3.x.js

```diff
@@ -0,0 +1,8762 @@
+declare module 'typescript' {
```

Comment:
where did this come from? doesn't have the usual flow-typed digest on top 🤔

Reply:
https://github.com/flow-typed/flow-typed/blob/master/definitions/npm/typescript_v3.3.x/flow_v0.42.x-/typescript_v3.3.x.js